### ISSUE TITLE
Commit message trimmed

### ISSUE THREAD

**Issue by poilu119** [mhutchie/vscode-git-graph#227]
The commit message shown at git graph has been trimmed.
Are there any way to show message not trimmed ? (config or settings)

Thank you.

src/dataSource
```typescript
public getTagDetails(repo: string, tagName: string): Promise<GitTagDetailsData> {
		return this.spawnGit(['for-each-ref', 'refs/tags/' + tagName, '--format=' + ['%(objectname)', '%(taggername)', '%(taggeremail)', '%(taggerdate:unix)', '%(contents)'].join(GIT_LOG_SEPARATOR)], repo, (stdout) => {
			let data = stdout.split(GIT_LOG_SEPARATOR);
			return {
				tagHash: data[0],
				name: data[1],
				email: data[2].substring(data[2].startsWith('<') ? 1 : 0, data[2].length - (data[2].endsWith('>') ? 1 : 0)),
				date: parseInt(data[3]),
				message: data[4].trim().split(EOL_REGEX).join('\n'),
				error: null
			};
		}).then((data) => {
			return data;
		}).catch((errorMessage) => {
			return { tagHash: '', name: '', email: '', date: 0, message: '', error: errorMessage };
		});
	}
```

**Comment by mhutchie** [comment]
There are currently no settings to disable trimming.

I'd be happy to implement it if I understood the motivation behind your request.

Why wouldn't you want the tags message to be trimmed (as trimming only removes leading & trailing white-space characters)?

**Comment by poilu119** [comment]
Using vscode as git editor, it will replace tab into spaces at default.
Therefore doing trimming will lose the leading tab.

But I do think trimming only trailing white-space should be okay in this case.
(or change to other editor / make vscode don't convert tab into spaces)

**Comment by mhutchie** [comment]
Thanks for confirming that, I'll remove the trimming of leading white-space in the next release, and make some other improvements around how leading white-space is rendered in the UI. A beta release will be available in the next day for you to use.

**Comment by mhutchie** [comment]
This will be available in v1.20.0.

If you'd like to use it before the next release, you can download [v1.20.0-beta.0](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.20.0-beta.0), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).

**Comment by poilu119** [comment]
Thank you for the update.
It works perfect   : )
